StgPlugin/src/org/workcraft/plugins/stg/concepts/ConceptsTask.java

```diff
@@ -16,24 +16,41 @@
 public class ConceptsTask implements Task<ExternalProcessResult> {
 
     private final File inputFile;
+    private Object[] includeList;
 
     public ConceptsTask(File inputFile) {
         this.inputFile = inputFile;
     }
 
+    public ConceptsTask(File inputFile, Object[] includeList) {
+        this.inputFile = inputFile;
+        this.includeList = includeList;
+    }
+
     @Override
     public Result<? extends ExternalProcessResult> run(ProgressMonitor<? super ExternalProcessResult> monitor) {
         try {
             ArrayList<String> command = new ArrayList<>();
 
-//            String stackLocation = StgSettings.getConceptsFolderLocation() + "stack";
             command.add("stack");
             command.add("runghc");
             String translateLocation = DesktopApi.getOs().isWindows() ? "translate\\Main.hs" : "translate/Main.hs";
-            command.add(StgSettings.getConceptsFolderLocation() + translateLocation); //TODO: Make concepts folder location a setting
-            command.add(inputFile.getAbsolutePath());
+            command.add(StgSettings.getConceptsFolderLocation() + translateLocation); // TODO:
```

Comment:
This multi-line comment is ugly, please put is as a one-line comment before the corresponding code line.